docs/reference/commands/cluster.rst

```diff
@@ -0,0 +1,210 @@
+===========
+``cluster``
+===========
+
+Manage the MicroCeph cluster
+
+Usage:
+
+.. code-block:: none
+
+   microceph cluster [options]
```

Comment:
We use 'options' here but 'flags' elsewhere. Let's stick with 'options' everywhere.

Reply:
I would like to keep this consistent with the help command output -- and this uses the `[flags]` slug (from the library, hard to change in our code)

Would it be ok unify this to `[flags]`?

Reply:
Alright, let's go with **flags** everywhere, unless there is a subtle difference between a flag and an option that I'm not privy to?